plugins/woocommerce/src/Blocks/BlockTypesController.php

```diff
@@ -401,6 +401,17 @@ protected function get_widget_area_block_types() {
 			'ProductsByAttribute',
 			'ProductCategory',
 			'ProductTag',
+			'ProductFilters',
+			'ProductFilterStatus',
+			'ProductFilterPrice',
+			'ProductFilterPriceSlider',
+			'ProductFilterAttribute',
+			'ProductFilterRating',
+			'ProductFilterActive',
+			'ProductFilterRemovableChips',
+			'ProductFilterClearButton',
+			'ProductFilterCheckboxList',
+			'ProductFilterChips',
```

Comment:
Ah! I just spotted this AFTER review, please move the `ProductFilters*` above line 394 which says:

> // Below product grids are hidden from inserter however they could have been used in widgets.
> // Keep them for backward compatibility.

This was used specifically to separate and highlight product grid blocks within the list.

Reply:
Right, I updated the list to make it clearer in https://github.com/woocommerce/woocommerce/pull/57487/commits/fa9fe8ca38959f5248d5b93d5f4ba5ceff2935b9. Let me know what you think.

Reply:
Nice! I like additional section with legacy filters! 💪 Ship! 🚢